src/algebraic_geometry/prime_spectrum.lean

```diff
@@ -354,4 +355,22 @@ begin
     exact subset_zero_locus_vanishing_ideal t }
 end
 
+/-- The prime spectrum of a commutative ring is a compact topological space. -/
+instance : compact_space (prime_spectrum R) :=
+begin
+  apply compact_space_of_finite_subfamily_closed,
```

Comment:
Is it possible to make this proof more readable, by adding some comments about what you are doing, by replacing `rcases` by `obtain`, or things like that?

Reply:
I will see what I can do.

Reply:
Thanks, it really makes a difference!